verifiers/internal/gha/npm.go

```diff
@@ -377,8 +367,37 @@ func verifyProvenanceSubjectVersion(b *utils.TrustedBuilderID, att *SignedAttest
 	return nil
 }
 
+func (n *Npm) verifySubjectDigest(expectedHash string) error {
+	publishSubjects, err := subjectsFromAttestation(n.verifiedPublishAtt)
+	if err != nil {
+		return err
+	}
+
+	// 8 bit represented in hex, so 8/2=4.
+	bitLength := len(expectedHash) * 4
+	expectedAlgo := fmt.Sprintf("sha%v", bitLength)
+	if bitLength < 256 {
+		return fmt.Errorf("%w: expected minimum 256-bit. Got %d", serrors.ErrorInvalidHash, bitLength)
```

Comment:
nit: I think this is a more understandable error message.
```suggestion
		return fmt.Errorf("%w: expected minimum sha256, got %s", serrors.ErrorInvalidHash, expectedAlgo)
```